### PR TITLE
fixes: #544 Update the URL to reflect the page the 'User' is on and remove spreadsheet for additional pages

### LINKED ISSUE #544: Update the URL to reflect the page the 'User' is on and remove spreadsheet for additional pages
**Summary:** The individual tabs on the ‘Offices’ page should be on their own route. When you click on ‘Areas’ from the ‘Offices’ page the URL should reflect this so we can use direct links to the individual sections. 

**Location**: https://dev.smb-app.reapit.cloud/offices

- Each tab should be on it’s on own URL, currently all linking to ‘Offices’

Areas (route offices\areas)
Global Settings (route offices\globalsettings)
Integrations (route offices\integrations)

See example of what the URL should be for Areas: 
image

**Summary:** The spreadsheet should only appear on ‘Offices’. The other tabs/pages (Areas, Global Setting/Integrations) should be blank for now other than the heading.

See example of how the 'Areas' page should look: 
image

### PR DESCRIPTION
**Pull request by phmngocnghia**
fixes #544

### CODE REVIEW

File: packages/smb/src/components/pages/offices/offices.tsx [hunk 1]
```diff
@@ -1,53 +1,95 @@
...
-  const [tab, handleChangeTab] = React.useState('offices')
+  const { pathname } = useLocation()
+  const history = useHistory()
+
+  const handleChangeTab = React.useCallback(path => {
+    history.push(path)
+  }, [])
+
+  const tabConfigs = React.useMemo(
+    () =>
```
Comment (tanphamhaiduong) [inline packages/smb/src/components/pages/offices/offices.tsx]: Pls ensure that this function cover by test ?

File: packages/smb/src/components/pages/offices/offices.tsx [hunk 2]
```diff
@@ -1,53 +1,95 @@
...
+  })
 
 export type OfficesProps = {}
 
 export const Offices: React.FC<OfficesProps> = () => {
-  const [tab, handleChangeTab] = React.useState('offices')
+  const { pathname } = useLocation()
+  const history = useHistory()
+
+  const handleChangeTab = React.useCallback(path => {
```
Comment (tanphamhaiduong) [inline packages/smb/src/components/pages/offices/offices.tsx]: pls ensure that this function cover by test

File: packages/smb/src/components/pages/offices/offices.tsx [hunk 3]
```diff
@@ -1,53 +1,95 @@
 import * as React from 'react'
 import { H3, FlexContainerBasic, FlexContainerResponsive, TabConfig, Tabs } from '@reapit/elements'
+import { useLocation, matchPath, useHistory } from 'react-router-dom'
+import { Route } from 'react-router'
 import OfficesTab from '@/components/ui/offices-tab'
+
```
Comment (tanphamhaiduong) [inline packages/smb/src/components/pages/offices/offices.tsx]: why do we need blank here ?

File: packages/smb/src/components/pages/offices/offices.tsx (hunk 1 above)
Comment (phmngocnghia) [reply packages/smb/src/components/pages/offices/offices.tsx]: I have added a test case for this fn

File: packages/smb/src/components/pages/offices/offices.tsx (hunk 2 above)
Comment (phmngocnghia) [reply packages/smb/src/components/pages/offices/offices.tsx]: I have added a test case for this

File: packages/smb/src/components/pages/offices/offices.tsx (hunk 3 above)
Comment (phmngocnghia) [reply packages/smb/src/components/pages/offices/offices.tsx]: Blank space have been removed

**Pull request merged**